crossbeam-epoch/src/atomic.rs

```diff
@@ -149,6 +150,24 @@ impl<T> Atomic<T> {
     ///
     /// let a = Atomic::<i32>::null();
     /// ```
+    #[cfg(loom)]
+    pub fn null() -> Atomic<T> {
```

Comment:
The reason is loom doesn't support const fn, right?  Does it now (at version 0.4)?

Reply:
Well, sort of. Loom's `AtomicUsize::new` is simply not `const` because it has to interact with the loom "runtime". Unlike `AtomicUsize::new` from `std`, which is really just a constant literal, loom's version must register the `AtomicUsize` with the model so that it's execution pattern can be tracked.